draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -894,6 +894,10 @@ particular use case and the Key Binding is not present, does not fulfill the req
 Key Binding data is present in the SD-JWT, the Verifier will reject the
 presentation, as described in (#verifier_verification).
 
+### Key Binding JWT
+
+Issuer provided integrity protection of the SD-JWT payload and Disclosures are achieved by the signature on the Issuer-signed JWT and the digest values in the signed payload as described in (#sec-is-jwt) and (#sec-disclosures) respectively. The Key Binding JWT, defined in (#kb-jwt), serves exclusively as a mechanism for the Holder to demonstrate possession of the private key corresponding to the public key in the SD-JWT payload. As such, the signature on the Key Binding JWT does not cover other parts of the SD-JWT. In cases where it's desirable for the Holder's signature to convey more than a proof-of-possession, such as signing over the selected Disclosures to prove those were the Disclosures selected, the SD-JWT to be presented can be embedded in another JWT (as described in [Enveloping SD-JWTs](#enveloping)) or otherwise signed by the Holder via the application protocol delivering it.
```

Comment:
```suggestion
Issuer provided integrity protection of the SD-JWT input claim set and Disclosures is achieved by the signature on the Issuer-signed JWT that contains SD-JWT input claim set and the digest values of the Disclosures as described in (#sec-is-jwt) and (#sec-disclosures) respectively. The Key Binding JWT, defined in (#kb-jwt), serves exclusively as a mechanism for the Holder to demonstrate possession of the private key corresponding to the public key in the SD-JWT payload. As such, the signature on the Key Binding JWT does not cover other parts of the SD-JWT. In cases where it's desirable for the Holder's signature to convey more than a proof-of-possession, such as signing over the selected Disclosures to prove those were the Disclosures selected, the SD-JWT to be presented can be embedded in another JWT (as described in [Enveloping SD-JWTs](#enveloping)) or otherwise signed by the Holder via the application protocol delivering it.
```

I would play around with the first sentence a little bit, but otherwise looks good.

Reply:
Proposed change on Kristina's proposed change:
```suggestion
Issuer provided integrity protection of the SD-JWT input claim set and Disclosures is achieved by the signature on the Issuer-signed JWT that covers the SD-JWT input claim set and the digest values of the Disclosures as described in (#sec-is-jwt) and (#sec-disclosures), respectively. The Key Binding JWT, defined in (#kb-jwt), serves exclusively as a mechanism for the Holder to demonstrate possession of the private key corresponding to the public key in the SD-JWT payload. As such, the signature on the Key Binding JWT does not cover other parts of the SD-JWT. In cases where it's desirable for the Holder's signature to convey more than a proof-of-possession, such as signing over the selected Disclosures to prove those were the Disclosures selected, the SD-JWT to be presented can be embedded in another JWT (as described in [Enveloping SD-JWTs](#enveloping)) or otherwise signed by the Holder via the application protocol delivering it.
```

Reply:
I'm not onboard with "SD-JWT payload" being wholesale replaced "SD-JWT input claim set".

So please make a suggested change with the editorial improvements you'd like to see that doesn't also change terminology.